Gopkg.toml

```diff
@@ -2,7 +2,7 @@
   name = "github.com/boltdb/bolt"
 
 [[constraint]]
-  name = "github.com/docker/notary"
+  name = "github.com/theupdateframework/notary"
```

Comment:
don't you have to also rename all the imports?

Reply:
Yes, good point (though I'm not sure why CI passed anyway). Also had to bump the commit we pin so that it doesn't transitively import `docker/notary`.

Reply:
I think CI passed because we use a cache https://github.com/kolide/launcher/blob/6e3b4226054aaa432d58fba4f66871dd6b4f3786/.circleci/config.yml#L9 and the old dep is still there.

Reply:
Could also be because we still ended up with `docker/notary` in the `Gopkg.lock` (it passed for me locally as well) due to the transitive dependency. Anyway, thanks for pointing this out.